aot/src/main/java/com/dylibso/chicory/experimental/aot/AotMethodInliner.java

```diff
@@ -33,17 +33,11 @@ public void visit(
                         super.visit(
                                 version,
                                 Opcodes.ACC_FINAL | Opcodes.ACC_SUPER,
-                                internalClassName(className + "$AotMethods"),
+                                internalClassName(className + "AotMethods"),
                                 null,
                                 superName,
                                 null);
                     }
-
-                    @Override
-                    public void visitEnd() {
```

Comment:
uhm, can this be related?
https://github.com/dylibso/chicory/pull/864/files#diff-70a4b55908ddb8adcb32f6a6c2402df3f22c41eb764e0828dafa07fb48422ab8R587

Reply:
So this commit does the reverse of that one..  That was was making all extra classes nested, this commit makes all extra classes top level.

Reply:
Thanks for explaining, I was getting confused :sweat_smile: , this PR is a larger body of work, if you can get "makes all extra classes top level" into a separate PR we can merge that one early :+1:

Reply:
ok.. splitting out to another PR.